crates/amaru/src/bin/amaru/cmd/import_ledger_state.rs

```diff
@@ -653,7 +655,7 @@ fn import_stake_pools(
     updates: BTreeMap<PoolId, PoolParams>,
     retirements: BTreeMap<PoolId, Epoch>,
     era_history: &EraHistory,
-) -> Result<(), impl std::error::Error> {
+) -> Result<(), impl std::error::Error + 'static> {
```

Comment:
@stevana This, BTW, is the concise way of saying “give me the edition 2021 behaviour back”.